projects/ad5766-sdz/src/ad5766_core.c

```diff
@@ -41,8 +42,10 @@
 /***************************** Include Files **********************************/
 /******************************************************************************/
 #include <stdlib.h>
-#include "platform_drivers.h"
 #include "spi_engine.h"
+#include "spi_engine_private.h"
```

Comment:
This header should be private, for SPI engine only.

Reply:
Yes, but the offload transfer is handled separately from the generic spi engine driver, through the ad5766_core.

Please see the note in the PR description.